buddyboss/buddyboss-platform
# Fixed - On Create Group page cover image placeholder should only show when image is uploaded

### All Submissions:


Fixes #2088 .

### How to test the changes in this Pull Request:

1. Go to create group page
2. step to Cover photo tab
3. blank Cover image placeholder should not show

### Proof Screenshots or Video

https://prnt.sc/xt9t1k
https://prnt.sc/xt9toy

### Other information:
